### PR TITLE
[.NET] Turkish NumbersWithUnit (Temperature) support

### PR DESCRIPTION
**Pull request by LionbridgeCS**
Temperature support. 
Passing: 31
Skipped: 1

Work in progress as same YAML is included in #1703.  once merged this will be updated to ready for review.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs [hunk 1]
```diff
@@ -23,40 +23,37 @@ public static class NumbersWithUnitDefinitions
...
-            { @"Milimetrekare", @"milimetre kare|milimetrekare|mm2|mm^2|mm²" },
-            { @"Mikrometrekare", @"mikrometre kare|mikrometrekare|µm2|µm^2|µm²" },
-            { @"Nanometrekare", @"nanometre kare|nanometrekare|nm2|nm^2|nm²" },
-            { @"İnçkare", @"inç kare|inçkare|in2|in^2|in²" },
-            { @"Fitkare", @"fit kare|fitkare|ft2|ft^2|ft²" },
-            { @"Milkare", @"mil kare|milkare|mi2|mi^2|mi²" },
-            { @"Yardakare", @"yarda kare|yardakare|yd2|yd^2|yd²" },
-            { @"Ayakkare", @"ayakkare" },
-            { @"Akre", @"akre|ac" },
+            { @"Square kilometer", @"kilometrekare|km2|km^2|km²" },
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs]: Forms like "kilometre kare" are not common even in informal text?

File: .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs [hunk 2]
```diff
@@ -23,40 +23,37 @@ public static class NumbersWithUnitDefinitions
...
         };
       public static readonly IList<string> AmbiguousAreaUnitList = new List<string>
         {
             @"fitkare",
             @"ar",
             @"dönüm",
             @"ha",
-            @"a"
+            @"a",
+            @"daa"
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs]: Out of curiosity, what's "daa" ambiguous with?

**Comment by tellarin** [comment]
@LionbridgeCS #1703 has been merged.

File: .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs (hunk 1 above)
Comment (LionbridgeCS) [reply .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs]: @tellarin they can be used in informal settings.

File: .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs (hunk 2 above)
Comment (LionbridgeCS) [reply .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs]: some company names are abbreviated as "daa"

**Comment by LionbridgeCS** [comment]
@tellarin: 'Draft' is removed, we will update to resolve the expected merge conflicts :)

File: .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs (hunk 1 above)
Comment (tellarin) [reply .NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs]: Then I think support for it should be added back.

**Comment by LionbridgeCS** [comment]
@tellarin thank you! yes we will add the definitions file along with a couple of cases.

**Pull request merged**